packages/components/src/FilterBar/FilterBar.scss

```diff
@@ -83,6 +83,11 @@ $tc-filter-bar-width: 250px !default;
 	background-color: $tc-filter-bar-highlight-color;
 }
 
+:global(.tc-list-toolbar) .filter {
+	margin-top: 9px;
+	margin-bottom: 9px;
```

Comment:
`9px` looks like a bit magic! Are guidelines referring to? I can't reach it.
At least, could you use a variable with `rm` unit, please?